src/routes/github/github-oauth-router.ts

```diff
@@ -16,7 +16,12 @@ const scopes = ["user", "repo"];
 const callbackPath = "/callback";
 
 const getRedirectUrl = async (req, res, state) => {
-	const { baseUrl } = req;
+	const basePaths = req.originalUrl.split("/");
+	let baseUrl  = basePaths.length > 0 ? `/${basePaths[1]}` : req.originalUrl;
+	if (res.locals?.gitHubAppConfig?.uuid) {
+		baseUrl += `/${res.locals.gitHubAppConfig.uuid}`;
+	}
```

Comment:
Since `GithubAuthMiddleware` is now bind with route, so `baseUrl` contains full path. 
For e.g. for route `/github/configuration`, earlier it `baseUrl` contains `/github`, now it has `/github/configuration`

Reply:
Not sure I follow.... Has this been tested in a test?